### ISSUE TITLE
Add U2F support.

### ISSUE THREAD

**Issue by ismell** [devgeeks/Encryptr#196]
Would love to do 2FA using U2F. Specifically a YubiKey.

**Comment by devgeeks** [comment]
Yeah, I have one of the new GitHub branded ones one the way. I will be looking into it to see what's possible.

**Comment by helveticade** [comment]
This would be awesome to support USB on Desktop and NFC on Android. 👌

**Comment by devgeeks** [comment]
:+1:

**Comment by sarndt** [comment]
same here, please add U2F 2FA :)

**Comment by f-edwards** [comment]
same! would love U2F

**Comment by Tormen** [comment]
Same here.
The ticket is open soon 2 years.
It would be nice to get a feedback from @spideroak on if this is something you can envision, or you won't implement.

**Comment by clounie** [comment]
Pretty ridiculous to be a password manager and not support hardware mfa in 2018.

**Comment by pabloi** [comment]
Same. Is there any timeline for this?
